src/components/Profile/ProfileInfo/ProfileData/ProfileData.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+
+import Contact from './Contact';
+
+const ProfileData = (props) => {
+  const { profile, isOwner, goToEditMode } = props;
+  return (
+    <div>
```

Comment:
Consider breaking this JSX into separate functions.

Reply:
/fxd